package.json

```diff
@@ -1,19 +1,41 @@
 {
   "name": "svelte-app",
   "version": "1.0.0",
+  "jest": {
+    "transform": {
+      "^.+\\.svelte$": "svelte-jester"
```

Comment:
If jest is configured for svelte, can we write svelte tests rather than js tests?

Reply:
I think the unit test is confirmation of what I was originally questioning here since the unit test actually does a svelte render and verifies attributes in js.